src/kudu/rpc/serialization.cc

```diff
@@ -137,7 +137,11 @@ Status ParseMessage(const Slice& buf,
   // Protobuf enforces a 64MB total bytes limit on CodedInputStream by default.
   // Override this default with the actual size of the buffer to allow messages
   // larger than 64MB.
+  #ifdef FBCODE_FIXES
+  in.SetTotalBytesLimit(buf.size());
+  #else
   in.SetTotalBytesLimit(buf.size(), -1);
+  #endif
```

Comment:
Later version of protobuf removed the second parameter, which was named `warning_threshold`. If we look at the third_party version inside kuduraft, the version of protobuf already simple discards that param without doing anything with it.

I can't just remove it here without breaking builds, so I added a FBCODE_FIXES flag so that in fbcode we skip the second param.

Reply:
Looked at this and our tp2 library already has something else which looks at protobuf version instead, which is an objectively better solution. I'll use that instead.